src/evm/types.ts

```diff
@@ -1,15 +1,6 @@
-import type { Chain as ViemChain } from "viem";
+import type { ChainBase } from "@src/shared/types";
```

Comment:
Will rename this to `ChainCommon` to eliminate the possibility of confusion with the Base EVM chain